openapi-spec/swagger.yaml

```diff
@@ -1656,13 +1730,54 @@ components:
         pub_key_type:
           type: string
           enum: ['ed25519', 'ecdsa', 'rsa']
+    CLIAccessInfoResponse:
+      required:
+        - host
+        - port
```

Comment:
In the response port is mentioned as required!

Reply:
@joseph-v plz remove port from required in response

Reply:
As discussed, response require port which can be None or valid number to be compliant.